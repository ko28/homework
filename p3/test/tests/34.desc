Unalias with too many arguments
