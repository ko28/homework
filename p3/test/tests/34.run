./mysh tests/34.in
