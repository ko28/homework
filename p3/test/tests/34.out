alias ls /bin/ls
unalias ls ls ls ls
unalias ls ls
alias
ls /bin/ls
exit
